kernel-boot/CMakeLists.txt

```
@@ -41,6 +41,9 @@ rdma_subst_install(FILES "persistent-ipoib.rules.in"
   DESTINATION "${CMAKE_INSTALL_DOCDIR}"
   PERMISSIONS OWNER_WRITE OWNER_READ GROUP_READ WORLD_READ)
 
+install(FILES "rdma_topo"
```

```suggestion
install(PROGRAMS "rdma_topo"
```

From https://cmake.org/cmake/help/latest/command/install.html#programs:
> The PROGRAMS form is identical to the FILES form except that the default permissions for the installed file also include OWNER_EXECUTE, GROUP_EXECUTE, and WORLD_EXECUTE. This form is intended to install programs that are not targets, such as shell scripts.